rholang/src/test/scala/coop/rchain/rholang/ProcGen.scala

```diff
@@ -169,15 +158,15 @@ object ProcGen {
     } yield new PNew(seqToJavaCollection[ListNameDecl, NameDecl](seqNameDecl), proc)
   }
 
-  private def procGen(procGens: Seq[State => Gen[Proc]], state: State): Gen[Proc] =
+  private def procGen(procGens: NonEmptyList[State => Gen[Proc]], state: State): Gen[Proc] =
     if (state.height > 0) {
       val newState = state.decrementHeight
       oneOf(procGens.map(_.apply(newState)))
     } else
       pnilGen
 
-  private val processContextProcs: Seq[State => Gen[Proc]] =
-    Seq(pgroundGen, pparGen, psendGen, pnewGen, pevalGen)
+  private val processContextProcs: NonEmptyList[State => Gen[Proc]] =
+    NonEmptyList.fromListUnsafe(List(pgroundGen _, pparGen _, psendGen _, pnewGen _, pevalGen _))
```

Comment:
Why not use `NonEmptyList.of(head: A, tail: A*)` here?

Reply:
True.